Client/Application/AppDelegate+Experiments.swift

```diff
@@ -10,6 +10,14 @@ import Foundation
 
 extension AppDelegate {
     func initializeExperiments() {
+        // We intialize the generated FxNimbus singleton very early on with a lazily
+        // constructed singleton.
+        FxNimbus.shared.initialize(with: { Experiments.shared } )
+        // We also make sure that any cache invalidation happens after each applyPendingExperiments().
+        NotificationCenter.default.addObserver(forName: .nimbusExperimentsApplied, object: nil, queue: nil) {
```

Comment:
```suggestion
        NotificationCenter.default.addObserver(forName: .nimbusExperimentsApplied, object: nil, queue: nil) { notification in
```

Pretty sure the missing argument is the reason this is failing on BR. I haven't worked with block-based notifs too much, so let's see. haha

What I'm curious about is whether or not we need to remove the observer?